pkg/validation/validate_cluster.go

```diff
@@ -267,7 +266,7 @@ func (v *ValidationCluster) validateNodes(cloudGroups map[string]*cloudinstances
 				continue
 			}
 
-			role := util.GetNodeRole(node)
+			role := strings.ToLower(string(cloudGroup.InstanceGroup.Spec.Role))
```

Comment:
Rather than performing this `ToLower` could we not change the `if n.Role == "..."` comparisons lower down to something like `if cloudGroup.InstanceGroup.Spec.Role == kops.InstanceGroupRoleMaster` instead as we already do with bastion role checks on line 254?

Reply:
The value gets returned to the caller in the `ValidationNode` struct (and the caller makes it visible to the user), so has to be lowercased in order to not do an API break.